app/components/Sidebar/components/Collections.js

```diff
@@ -32,17 +33,22 @@ function Collections({ onCreateCollection }: Props) {
 
   React.useEffect(() => {
     async function load() {
-      if (!collections.isLoaded && !isFetching) {
+      if (!collections.isLoaded && !isFetching && !fetchError) {
         try {
           setFetching(true);
           await collections.fetchPage({ limit: 100 });
+        } catch (error) {
+          ui.showToast(error.message + t(", try again?"), {
+            type: "error",
+          });
+          setFetchError(true);
```

Comment:
```suggestion
          ui.showToast(t("Collections could not be loaded, please reload the app"), {
            type: "error",
          });
          setFetchError(error);
```

The error message isn't guaranteed to make sense to the end user, probably clearer to just hard code a message here. "Try again?" doesn't make as much sense for an action that was automatic to begin with.

Reply:
Yes Yes, taking note that it was automatic, tru will not definitely make sense.